src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -145,7 +145,6 @@
   <!-- Sources -->
   <ItemGroup>
       <Compile Include="resources\SR.extra.cs" />
-      <Compile Include="$(CoreClrProjectRoot)\src\System.Private.CoreLib\src\System\SR.cs" />
   </ItemGroup>
   <ItemGroup>
       <Compile Include="$(BclSourcesRoot)\Microsoft\Win32\UnsafeNativeMethods.cs" />
```

Comment:
BTW: This file seems to be all dead code.

Reply:
Thanks, deleted.